magma/uniquification.py

```diff
@@ -27,6 +27,8 @@ class _HashStruct:
 
 def _make_hash_struct(definition):
     repr_ = repr(definition)
+    for s, args, st, prefix in definition._context_._inline_verilog:
```

Comment:
another option is to make these explicit fields of the HashStruct. might be cleaner and more transparent/maintainable

Reply:
Looks good.